README.md

```diff
@@ -2,9 +2,9 @@
   <a href="https://js.devexpress.com/" title="DevExtreme-Quill">DevExtreme Quill</a>
 </h1>
 
-Devextreme-Quill is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
+The **DevExtreme Quill** is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
 
-Comparing to the current stable version of Quill, Devextreme-Quill provides basic table support and improve working with lists.
+Comparing to the current stable version of Quill, the **DevExtreme Quill** supports basic table operations and brings enhancements to lists rendering. 
```

Comment:
"Comparing"? Why do you need to use a gerund here?

Reply:
```suggestion
**DevExtreme Quill** supports basic table operations and enhances lists rendering. 
```

Reply:
To be honest, I just don't know how to say the same thing without gerund. We need to clarify, why we have our own fork, so I want to compare our fork's functionality with the original Quill.

Reply:
I suggested an alternative version